src/pages-and-resources/proctoring/Settings.test.jsx

```diff
@@ -215,186 +216,198 @@ describe('ProctoredExamSettings', () => {
           proctoring_escalation_email: 'test@example.com',
           create_zendesk_tickets: true,
         },
-        available_proctoring_providers: ['software_secure', 'proctortrack', 'mockproc'],
+        available_proctoring_providers: ['software_secure', 'proctortrack', 'mockproc', 'lti_external'],
         course_start_date: '2070-01-01T00:00:00Z',
       });
 
+      axiosMock.onPatch(
+        ExamsApiService.getExamConfigurationUrl(defaultProps.courseId),
+      ).reply(204, {});
+
       axiosMock.onPost(
         StudioApiService.getProctoredExamSettingsUrl(defaultProps.courseId),
       ).reply(200, {});
 
       await act(async () => render(intlWrapper(<IntlProctoredExamSettings {...defaultProps} />)));
     });
 
-    it('Creates an alert when no proctoring escalation email is provided with proctortrack selected', async () => {
-      await waitFor(() => {
-        screen.getByDisplayValue('proctortrack');
+    proctoringProvidersRequiringEscalationEmail.forEach(provider => {
+      it(`Creates an alert when no proctoring escalation email is provided with ${provider} selected`, async () => {
```

Comment:
I moved the majority of tests that are marked as deleted below into this for loop to test both Proctortrack and LTI-based providers.

The one difference is I corrected the test name `Has no error when invalid proctoring escalation email is provided with proctoring disabled` to `Has no error when empty proctoring escalation email is provided with proctoring disabled`.

Reply:
ahh good idea 👍